pkg/registry/generic/etcd/etcd.go

```diff
@@ -271,10 +272,21 @@ func (e *Etcd) Update(ctx api.Context, obj runtime.Object) (runtime.Object, bool
 	if err != nil {
 		return nil, false, err
 	}
+	// If the object specified by the user does not have a resource version,
```

Comment:
Yeah, something like that.  That allows us to add something to resttest that you indeed fail if that is false.

----- Original Message -----
> > @@ -271,10 +272,21 @@ func (e *Etcd) Update(ctx api.Context, obj
> > runtime.Object) (runtime.Object, bool
> >  	if err != nil {
> >  		return nil, false, err
> >  	}
> > +	// If the object specified by the user does not have a resource version,
> 
> I can add a AllowUpdateWithoutResourceVersion() like AllowCreateOnUpdate()
> (https://github.com/GoogleCloudPlatform/kubernetes/blob/5520386b180d3ddc4fa7b7dfe6f52642cc0c25f3/pkg/api/rest/update.go#L35)
> in RESTUpdateStrategy?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/10074/files#r32854948